charts/ocis/templates/proxy/config.yaml

```diff
@@ -14,7 +14,7 @@ data:
         policy: ocis
     {{- with $.Values.features.quotas.roles }}
     role_quotas:
-      {{- toYaml . | nindent 8 }}
+      {{- toYaml . | nindent 4 }}
```

Comment:
Sure about the 4? I would say it should be 6

Reply:
actually this should be kept unchanged, will revert it.

Reply:
This is what I got when I installed the development-install example:

`kubectl get -o yaml configmap -n ocis proxy-config`

```yaml
apiVersion: v1
data:
  csp.yaml: |
    ---
    directives:
      child-src:
        - '''self'''
      connect-src:
        - '''self'''
      default-src:
        - '''none'''
      font-src:
        - '''self'''
      frame-ancestors:
        - '''self'''
      frame-src:
        - '''self'''
        - 'blob:'
        - https://embed.diagrams.net/
      img-src:
        - '''self'''
        - 'data:'
        - 'blob:'
      manifest-src:
        - '''self'''
      media-src:
        - '''self'''
      object-src:
        - '''self'''
        - 'blob:'
      script-src:
        - '''self'''
        - '''unsafe-inline'''
      style-src:
        - '''self'''
        - '''unsafe-inline'''
  proxy.yaml: |
    ---
    policy_selector:
      static:
        policy: ocis
kind: ConfigMap
metadata:
  annotations:
    meta.helm.sh/release-name: ocis
    meta.helm.sh/release-namespace: ocis
  creationTimestamp: "2024-07-12T09:59:47Z"
  labels:
    app.kubernetes.io/instance: ocis
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: ocis
    app.kubernetes.io/version: 6.1.0
    helm.sh/chart: ocis-0.7.0
  name: proxy-config
  namespace: ocis
  resourceVersion: "71690"
  uid: 67cae78b-2914-4ee1-9c91-294443a6de45
```

Reply:
Hmm but unchanged (so 4) also sounds wrong to me if I look at the indentation.

Reply:
It was reverted in https://github.com/owncloud/ocis-charts/pull/636/commits/8d67428254c2fdfbeae717b4b3e1d78571a3148b